doc/src/stdlib/iterators.md

```diff
@@ -12,4 +12,6 @@ Base.Iterators.repeated
 Base.Iterators.product
 Base.Iterators.flatten
 Base.Iterators.partition
+Base.Iterators.filter
```

Comment:
Is there some reason that `Iterators.filter` was not listed here previously?  It seemed like an oversight.

Also, the ordering of methods here seems kind of random; is it automatically sorted?

Reply:
I agree, I think it was an oversight. Documenter doesn't do any sorting of methods—they appear in the output in the same order as you write them AFAIK.

Reply:
Actually, now that I look more closely at the order, it seems like they have been somewhat topically sorted, so I think the order is fine as-is.